fjord/mailinglist/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from fjord.mailinglist.models import MailingList
+
+
+class MailingListAdmin(admin.ModelAdmin):
+    list_display = ('name', 'members')
+
+
+admin.site.register(MailingList, MailingListAdmin)
```

Comment:
You might have to explicitly pass the admin site instance if you're monkeypatching the admin site with a new one, depending on the order of imports and monkeypatching.